src/main/frontend/app/App.js

```diff
@@ -0,0 +1,63 @@
+import React, {Component} from 'react';
+import ReviewFormContainer from './containers/ReviewFormContainer'
+
+class App extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      reviews: []
+    }
+
+    this.addNewReview = this.addNewReview.bind(this)
+  }
+
+  componentDidMount() {
+    fetch('/api/v1/reviews')
+    .then(response => {
+      return response.json()
+    })
+    .then(reviews => {
+      this.setState( {reviews: reviews} )
+    })
+  }
+
+  addNewReview(formPayload) {
+    fetch("/api/v1/reviews", {
+      method: "POST",
+      headers: {"Content-Type": "application/json"},
+      credentials: "same-origin",
+      body: JSON.stringify(formPayload)
+    })
+      .then(response => response.json())
+      .then(body => {
+        this.setState( {reviews: [...this.state.reviews, body]} )
+      })
+      .catch(error => console.log(`Error in fetch: ${error.message}`))
+  }
+
+  render(){
+    let reviewsArray = this.state.reviews.map(review => {
+      return (
+        <div>
```

Comment:
this markup does not conform to W3C standards. You cannot have a `<div>` as an immediate parent to `<li>` - `<li>` tags must be direct descendants of `<ul>` or `<ol>`